### PR TITLE
extension of PR #2851

### PR DESCRIPTION
**Pull request by kawaii**
#2851

@JordanMussi asked me to extend my PR allowing access to the contact page when the board is closed, but since I'd deleted the original forked repository I couldn't figure out the git magic to extend the original PR, so I've created this new one. #2851 can be closed.

### CODE REVIEW

File: global.php [hunk 1]
```diff
@@ -966,6 +966,7 @@
 		'logout',
 	),
 	'captcha.php',
+	'contact.php,
```
Comment (Shade-) [inline global.php]: Missing a single quote at the end of the string, just before the coma.

File: global.php [hunk 2]
```diff
@@ -1008,6 +1009,7 @@
 		'resetpassword',
 	),
 	'captcha.php',
+	'contact.php,
```
Comment (Shade-) [inline global.php]: Same here.

**Comment by kawaii** [comment]
@Shade- fixed, sorry I was tired. ;)

**Comment by JordanMussi** [comment]
Thanks @kawaii! 😀

**Pull request merged**